README.md

```diff
@@ -0,0 +1,94 @@
+# sysext-bakery: Recipes for baking systemd-sysext images
+
+[Systemd-sysext images](https://www.freedesktop.org/software/systemd/man/systemd-sysext.html) are overlay images for `/usr`, allowing to extend the base OS with custom (static) binaries.
+Flatcar Container Linux as an OS without a package manager is a good fit for extension through systemd-sysext.
+The tools in this repository help you to create your own sysext images bundeling software to extend your base OS.
+The current focus is on Docker and containerd, contributions are welcome for other software.
+
+## Systemd-sysext on Flatcar
+
+The `NAME.raw` sysext images (or `NAME` sysext directories) can be placed under `/etc/extensions/` or `/var/lib/extensions` in Flatcar to be activated on boot.
+While systemd-sysext images are not really meant to also include the systemd service, Flatcar ships `ensure-sysext.service` as workaround to automatically load the image's services.
+It reloads the unit files from disk and reevaluates `multi-user.target`, `sockets.target`, and `timers.target`, making sure your enabled systemd units run.
+This service is bound to `systemd-sysext.service` which activates the sysext images on boot.
+At runtime executing `systemctl restart systemd-sysext` will reload the sysext images and start the services.
+A manual `systemd-sysext refresh` is not recommended.
+
+The compatibility mechanism of sysext images requires a metadata file in the image under `usr/lib/extension-release.d/extension-release.NAME`.
+It has to contain a matching OS `ID`, and either a matching `VERSION` or `SYSEXT_LEVEL`.
+Since the rapid release cycle and automatic updates of Flatcar Container Linux make it hard to rely on particular OS libraries by specifying a dependency of the sysext image to the OS version, it is not recommended to match by `VERSION`.
```

Comment:
```suggestion
Since the rapid release cycle and automatic updates of Flatcar Container Linux makes it hard to rely on particular OS libraries by specifying a dependency of the sysext image to the OS version, it is not recommended to match by `VERSION`.
```

Reply:
I think "make" is correct for the plural "A and B"?

Reply:
You're right, I was only considering the "rapid release cycle". :)